internal/backend/runtime/omni/controllers/omni/machine_status_test.go

```diff
@@ -306,7 +309,8 @@ func (suite *MachineStatusSuite) TestMachineSchematic() {
 				},
 			},
 			expected: &specs.MachineStatusSpec_Schematic{
-				Id: "7d79f1ce28d7e6c099bc89ccf02238fb574165eb4834c2abf2a61eab998d4dc6",
+				Id:               "7d79f1ce28d7e6c099bc89ccf02238fb574165eb4834c2abf2a61eab998d4dc6",
```

Comment:
probably add a comment showing the yaml data used to produce this input

Reply:
It's not yaml, but the definition of the schematic is right above these lines.

Reply:
i missed the definitions, all good here :+1: